### ISSUE TITLE
After training, we've got the log.json which is about loss.

### ISSUE THREAD

**Issue by BaeSukkyoung** [open-mmlab/mmdetection#4200]
When I look at the log.json file, I can see the various kind of loss such as loss_cls, loss_bbox, loss_mask.

Is it right that loss = a*loss_cls(rpn_loss_cls+s0.loss_cls+s1.loss_cls+s2.loss_cls)+
b*loss_bbox(rpn_loss_bbox+s0.loss_bbox+s1.loss_bbox+s2.loss_bbox)+
c*loss_mask(s0.loss_mask+s1.loss_mask+s2.loss_mask) ? (In here, a, b, c is weight of each loss)

If it is right, how can I approve this ?

**Comment by ZwwWayne** [comment]
The loss is just the summation of each loss item. The weight of each loss item has already be used when calculating each single loss item. But I do not get what you want to approve.
